ext/opentelemetry-ext-grpc/src/opentelemetry/ext/grpc/__init__.py

```diff
@@ -21,7 +21,7 @@
 from opentelemetry.ext.grpc.version import __version__
 
 
-def client_interceptor(tracer_provider=None):
+def client_interceptor(tracer_provider=None, meter=None):
```

Comment:
How will the metrics be exposed if the instrumentor is auto-instrumented? There's no way to pass in/access a meter if the instrumentor is instantiated through an entry point.

Reply:
I don't think there is a good way to have the metrics be created on auto instrumentation, since there is no global meterprovider + interval + exporter we can attach to. The meter has to be passed in somehow, and if auto-instrumentation can't pass anything in, then we can't add metrics.

Reply:
Right. In the above suggestion for export + interval, this only works on explicit instantiation. Seems like we can either expose an API for the user to manually indicate to start exporting metrics, or we can use a default exporter + export interval. Both methods are kind of not great IMO, it either requires the user to know about the implementation/existence of metrics in the first place (and they have to manually call start_pipeline) or it forces them to use an exporter/export interval that is not configurable. Might be something to bring up during the SIG.